data/src/test/java/com/teamagam/gimelgimel/data/message/repository/cache/room/Converters/ConverterTest.java

```diff
@@ -0,0 +1,55 @@
+package com.teamagam.gimelgimel.data.message.repository.cache.room.Converters;
+
+import com.teamagam.gimelgimel.domain.base.sharedTest.BaseTest;
+import java.util.Date;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+
+public class ConverterTest extends BaseTest {
+
+  private static final int EQUALS_VALUE = 0;
+
+  @Test
+  public void convertDate_expectTicks() throws Exception {
+    // Arrange
+    long expected = 636289654156321489L;
+    Date d = new Date(expected);
+
+    // Act
+    long result = DateConverter.dateToTimestamp(d);
+
+    // Assert
+    assertEquals(result, expected);
+  }
+
+  @Test
+  public void convertTicks_expectValidDate() throws Exception {
+    // Arrange
+    long ticks = 636289654156321489L;
+    Date expected = new Date(ticks);
+
+    // Act
+    Date result = DateConverter.fromTimestamp(ticks);
+
+    // Assert
+    assertEquals(EQUALS_VALUE, result.compareTo(expected));
```

Comment:
R u sure assertEquals doesn't use compareTo?
If not, I find `assertThat(result.compareTo(expected), is(0))` more readable.